tests/app.test.js

```diff
@@ -0,0 +1,60 @@
+
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+chai.use(chaiHttp);
+const { assert } = chai;
+const app = require('../lib/app');
+const random = require('../lib/random.facts.js');
+
+describe('server', () => {
+    const request = chai.request(app);
+
+    it('says hello stranger when no name specified', done => {
+        request.get('/greeting')
+            .end((err, res) => {
+                if (err) return done(err);
+                assert.equal(res.text, 'hello stranger');
+                done();
+            });
+    });
+
+    it('says hello monica when name monica is specified', done => {
+
+        request.get('/greeting/monica')
+            // request.get('/greeting/jane?salutation=yo')
```

Comment:
Try to do a little more cleanup before turning in code - delete commented-out code, make your use of whitespace consistent, etc.